ps3_hangman.py

```diff
@@ -0,0 +1,133 @@
+# Hangman game
```

Comment:
This seems like an assignment from a problem set. I may be interested in putting exercises in this repo for the future, but putting it at the root of the repository seems to imply that all exercises should be placed at the root as well.

Exercises can be placed somewhere else.